src/Camelot/Views/Main/FilesPanelView.xaml.cs

```diff
@@ -14,7 +14,8 @@ namespace Camelot.Views.Main
 {
     public class FilesPanelView : UserControl
     {
-        private FilesPanelViewModel ViewModel => (FilesPanelViewModel) DataContext;
+        private DataGrid FilesDataGrid => this.FindControl<DataGrid>("FilesDataGrid");
```

Comment:
Please use this property in `ClearSelection` method too and add empty line after this one just to keep everything consistent :smiley: